portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,104 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+//HELPER_FUNCTION
+/**Create a typewriter from a given div */
+function typify(div_id){
+  //get the div
+  var div = document.getElementById(div_id);
+  //turn into typewriter object
+  return new Typewriter(div, {
+    loop: false,
+    delay: 10,
+    autoStart: true,
+    cursor: '',
+    strings: []
+  });
 }
+
+// TEXT
+/**Heading text */
+const heading_text = "<div class = 'row'> <h1> Obi Abii's Portfolio </h1> </div>"; 
+
+/**About me blurb */
+const about_me_text = "<div class = 'row'> <p>Hi, My name is Obi and this is my <span class='emphasis'>portfolio! <br></span></p>\
+<p>I'm a CS Major at Cornell, currently interning at Google for the Summer of 2020. My main career focus is <span class='emphasis'>software engineering</span>. \
+I am particularly interested in <span class='emphasis'>Machine learning.</span></p></div>";
```

Comment:
For our HTML in JS, let's try using the template literals in JS? https://google.github.io/styleguide/jsguide.html#features-strings-template-strings. See also https://google.github.io/styleguide/jsguide.html#features-strings-no-line-continuations, no \ continuations.

Below too